scripts/auto-rebase/rebase.sh

```diff
@@ -698,7 +713,7 @@ rebase_to() {
     fi
 
     update_images
-    if [[ -n "$(git status -s pkg/release)" ]]; then
+    if [[ -n "$(git status -s pkg/release packaging/crio.conf.d)" ]]; then
```

Comment:
What's going on here?

Reply:
Something I discovered testing manually. If no changes are in `pkg/release` we won't `git add packaging/crio.conf.d` resulting in not committing it - I think it's just pause image but still